### ISSUE TITLE
train_classification.py error

### ISSUE THREAD

**Issue by 3bobo** [fxia22/pointnet.pytorch#31]
@fxia22 I can run with `train_segmentation.py`, but when I try to run `train_classification.py`, the data is all zero like this:
> 
> [0: 0/15990] train loss: 0.000000 accuracy: 1.000000
> [0: 0/15990] test loss: 0.000000 accuracy: 0.000000
> [0: 1/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 2/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 3/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 4/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 5/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 6/15990] train loss: 0.000000 accuracy: 1.000000
> [0: 7/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 8/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 9/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 10/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 10/15990] test loss: 0.000000 accuracy: 0.000000
> [0: 11/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 12/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 13/15990] train loss: 0.000000 accuracy: 1.000000
> [0: 14/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 15/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 16/15990] train loss: 0.000000 accuracy: 1.000000
> [0: 17/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 18/15990] train loss: 0.000000 accuracy: 0.000000
> [0: 19/15990] train loss: 0.000000 accuracy: 1.000000
> [0: 20/15990] train loss: 0.000000 accuracy: 0.000000
>
because my computer's gpu is not enough strong, I set batch_size to 1, does is matter?
thanks for your help.

**Comment by 3bobo** [comment]
it seemed the zero was caused by my batch_size was too small.
